addons/web/static/tests/webclient/actions/client_action_tests.js

```diff
@@ -518,6 +518,57 @@ QUnit.module("ActionManager", (hooks) => {
         );
     });
 
+    QUnit.test("test display_notification client action with multiple links", async function (assert) {
+        assert.expect(8);
+        const webClient = await createWebClient({ serverData });
+        await doAction(webClient, 1);
+        assert.containsOnce(webClient, ".o_kanban_view");
+
+        const links = [
+            { label: "test1 <R&D>", url: "#action={action.id}&id={order.id}&model=purchase.order" },
+            { label: "test2 <R&D>", url: "#action={action.id}&id={order.id}&model=purchase.order" },
+            { label: "test3 <R&D>", url: "#action={action.id}&id={order.id}&model=purchase.order" },
+        ];
+
+        await doAction(webClient, {
+            type: "ir.actions.client",
+            tag: "display_notification",
+            params: {
+                title: "title",
+                message: "message %s <R&D>",
+                sticky: true,
+                links: links,
+            },
+        });
+        const notificationSelector = ".o_notification_manager .o_notification";
+        assert.containsOnce(
+            document.body,
+            notificationSelector,
+            "a notification should be present"
+        );
+        let notificationElement = document.body.querySelector(notificationSelector);
+        assert.strictEqual(
+            notificationElement.querySelector(".o_notification_title").textContent,
+            "title",
+            "the notification should have the correct title"
+        );
+        assert.strictEqual(
+            notificationElement.querySelector(".o_notification_content").textContent,
+            `message test1 <R&D>,test2 <R&D>,test3 <R&D> <R&D>`,
+            "the notification should have the correct message"
+        );
+        const notificationLinks = notificationElement.querySelector(".o_notification_content");
+        links.forEach((link, index) => {
```

Comment:
we don't really care about the number of lines in tests. In this case, I definitely prefer 3 `assert.strictEqual` lines that are easier to read that a forEach (which is also written on 3 lines btw).

Reply:
@aab-odoo Yeah, The Changes have been made, Thank you for your suggestion